### ISSUE TITLE
Licensee doesn't find license

### ISSUE THREAD

**Issue by gabrielittner** [cashapp/licensee#79]
I'm getting this error
```
com.freeletics.mad:navigator-runtime:0.3.0-alpha27
 - ERROR: Artifact declares no licenses!
```

however the license is in the [pom](https://repo.maven.apache.org/maven2/com/freeletics/mad/navigator-runtime/0.3.0-alpha27/navigator-runtime-0.3.0-alpha27.pom) and the previous [release](https://repo.maven.apache.org/maven2/com/freeletics/mad/navigator-runtime/0.3.0-alpha26/navigator-runtime-0.3.0-alpha26.pom) which looks the same worked just fine. My theory is that licensee gets confused by the newer release using Android's new multi variant publishing, even though that doesn't have any influence on the pom.

**Comment by JakeWharton** [comment]
Yeah sounds similar to #67, #58, and #30. We're using standard Gradle mechanisms to resolve the pom so something is going wrong. I just haven't had the time to really dig into it.

**Comment by JakeWharton** [comment]
Going to close as a duplicate of #30 which is fixed in 1.5.0. Please try and let me know if it resolves this. Otherwise we'll have to get a failing test case integrated.

**Comment by gabrielittner** [comment]
Can confirm that it's fixed 👍
